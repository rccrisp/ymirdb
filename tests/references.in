set a 1
set b 2
set c 3
snapshot
set c a b
snapshot
forward c
set c a a
forward c
backward a
checkout 1
forward c
backward a
bye
