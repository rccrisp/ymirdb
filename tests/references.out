> ok

> ok

> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> a, b

> ok

> a

> c

> ok

> nil

> nil

> bye
